dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/LinkageChecker.java

```diff
@@ -456,6 +456,13 @@ private boolean isMemberAccessibleFrom(
     String sourceClassName = classFile.getBinaryName();
     String targetClassName = symbol.getClassBinaryName();
 
+    // Skip references to Java runtime class. For example,
+    // com.sun.xml.internal.ws.api.BindingID$SOAPHTTPImpl
+    // https://github.com/GoogleCloudPlatform/cloud-opensource-java/issues/1599
+    if (classDumper.isSystemClass(targetClassName)) {
```

Comment:
Let's change this on source.

It's possible that target is a system class and we detect real problems.

Reply:
`com.sun.tools.internal.ws.wscompile.WsgenOptions` is loaded by `Class Loader for WsgenOptions:sun.misc.Launcher$AppClassLoader@18b4aac2`, not the extension class loader. How can we distinguish class from JDK and others?